meta/main.yml

```diff
@@ -8,7 +8,7 @@ galaxy_info:
   author: 'Robin Schneider'
   description: 'Manage and configure the Apache HTTP Server'
   license: 'GPL-3.0'
-  min_ansible_version: '2.1.5'
+  min_ansible_version: '2.2.0'
```

Comment:
Ah. I overlooked that you only changed the min version in in once place. It is contained in multiple files. I just added the missing docs to https://github.com/ypid/ypid-ansible-common/blob/master/bin/ansible-update-min-version

For the next time maybe just grep over the repo to find such things. The version number is in readme and docs.